app/src/features/sessionBook/components/SaveSessionButton/SaveSessionButton.tsx

```diff
@@ -38,20 +58,69 @@ export const SaveSessionButton = () => {
       });
   };
 
+  const handleSaveSession = () => {
```

Comment:
wrap this in a `useCallaback`. 
if we don't add attributes like `user`, `userAttributes` and `workspace.id`, we risk `saveDraftSession` getting an old closured value in cases when user changes auth state or workspaces.